src/main/scala/com/horizon/exchangeapi/AgbotsRoutes.scala

```diff
@@ -1575,11 +1575,12 @@ trait AgbotsRoutes extends JacksonSupport with AuthenticationSupport {
       complete({
         val nodeId = ident.creds.id      //somday: handle the case where the acls allow users to send msgs
         var msgNum = ""
+        val maxMessagesInMailbox = ExchConfig.getInt("api.limits.maxMessagesInMailbox")
+        val getNumOwnedDbio = if (maxMessagesInMailbox == 0) DBIO.successful(0) else AgbotMsgsTQ.getNumOwned(compositeId).result // avoid DB read for this if there is no max
         // Remove msgs whose TTL is past, then check the mailbox is not full, then get the node publicKey, then write the agbotmsgs row, all in the same db.run thread
-        db.run(AgbotMsgsTQ.getNumOwned(compositeId).result.flatMap({ xs =>
+        db.run(getNumOwnedDbio.flatMap({ xs =>
           logger.debug("POST /orgs/"+orgid+"/agbots/"+id+"/msgs mailbox size: "+xs)
```

Comment:
same comment for the debug message for mailbox size

Reply:
I'll fix this too...